.gitignore

```diff
@@ -38,3 +38,4 @@ po/gen-pot
 *.rej
 *.pem
 ui/pages/help/*/*.html
+uiTests/.env
```

Comment:
I think it would be better to place these tests under `/tests` so we have a single place to look for tests.

Maybe `/tests/ui`

Reply:
Also I don't think we need to add `kimchi_project` as part of the file structure (`uiTests/pages/kimchi_project/*.py`) as everything inside Kimchi repo should be only related to Kimchi.
I suggest to use `tests/ui/pages/*.py`

Reply:
The kimchi dir relates to the Page Objects concept:

https://page-objects.readthedocs.io/en/latest/